CHANGELOG.md

```diff
@@ -122,20 +414,20 @@ been added to each release, please refer to the [blog](https://blog.gitea.io).
   * Move add to hook queue for created repo to outside xorm session. (#7682) (#7675)
   * Show protection symbol if needed on default branch (#7660) (#7668)
   * Hide delete/restore button on archived repos (#7660)
-  * Fix bug on migrating milestone from github (#7665) (#7666) 
+  * Fix bug on migrating milestone from github (#7665) (#7666)
```

Comment:
Why these changes added here?

Reply:
Because of attom

Reply:
Are this on noicy?

Reply:
I can try to rebase ...

Reply:
But it only delete useles space at the end so this wasent something bad i thought ... @lunny